lib/elinux_plugins.dart

```diff
@@ -547,22 +599,23 @@ void RegisterPlugins(flutter::PluginRegistry* registry);
   _renderTemplateToFile(
     '''
 //
-// Generated file. Do not edit.
+//  Generated file. Do not edit.
 //
 
 // clang-format off
 
 #include "generated_plugin_registrant.h"
 
-{{#plugins}}
+{{#methodChannelPlugins}}
 #include <{{name}}/{{filename}}.h>
-{{/plugins}}
+{{/methodChannelPlugins}}
 
 void RegisterPlugins(flutter::PluginRegistry* registry) {
-{{#plugins}}
-  {{class}}RegisterWithRegistrar(
-      registry->GetRegistrarForPlugin("{{class}}"));
-{{/plugins}}
+{{#methodChannelPlugins}}
+  g_autoptr(FlPluginRegistrar) {{name}}_registrar =
+      fl_plugin_registry_get_registrar_for_plugin(registry, "{{class}}");
```

Comment:
I didn't check the latest official flutter codes yet, but I believe this `fl_plugin_registry_get_registrar_for_plugin` is not correct to be used in flutter-elinux because the plugin interfaces of the flutter-elinux is compatible with flutter desktop for Windows not Linux. Can you please refer to the plugin interface for Windows desktop? Also, please verify your code by using https://github.com/sony/flutter-elinux-plugins. This change will fail when you run the plugins such as video player for elinux.

Reply:
> plugin interface for Windows

https://github.com/flutter/flutter/blob/1dce0c7ffb9e9686876400d69a1d26985a43169e/packages/flutter_tools/lib/src/flutter_plugins.dart#L525-L528

Got it! I have fixed that